app/controllers/assignments_controller.rb

```diff
@@ -14,6 +15,10 @@ def index
   def new
     @week = Week.find(params[:week])
 
+    if Assignment.where(week: @week).count >= 2
```

Comment:
Does `2` appear anywhere else in the app? I think I'd pull this out into a constant in the assignment model (`ASSIGNMENTS_PER_WEEK = 2`).